scripts/test/ISISDirecInelasticConfigTest.py

```diff
@@ -69,21 +71,31 @@ def _set_up(self):
 
     def makeFakeSourceReductionFile(self,mcf,contents=None):
 
-        instr_name = mcf._user.instrument
+        all_instr_names = mcf._user.get_all_instruments()
+        all_files = []
+        for instr in all_instr_names:
+            instr_name = instr
 
-        file_path = os.path.join(self.UserScriptRepoDir,'direct_inelastic',instr_name.upper())
-        if not os.path.exists(file_path):
-            os.makedirs(file_path)
+            file_path = os.path.join(self.UserScriptRepoDir,'direct_inelastic',instr_name.upper())
+            if not os.path.exists(file_path):
+                os.makedirs(file_path)
         
-        file_name = mcf._sample_reduction_file(instr_name)
-        full_file = os.path.join(file_path,file_name)
-        if os.path.isfile(full_file):
-            os.remove(full_file)
-        fh=open(full_file,'w')
-        fh.write('#Test reduction file\n')
-        fh.write('Contents={0}'.format(contents))
-        fh.close()
-        return full_file
+            file_name = mcf._sample_reduction_file(instr_name)
+            all_files.append(file_name)
+            full_file = os.path.join(file_path,file_name)
+            if os.path.isfile(full_file):
+                os.remove(full_file)
+            fh=open(full_file,'w')
```

Comment:
good point, thanks -- unfortunately can not recall the exact form each time I deal with such issue in Python. Will try to use it next time.